python311Packages.huggingface-hub: 0.22.0 -> 0.22.2, python311Packages.transformers: 4.39.1 -> 4.39.2

GaetanLepage opened the pull request:
## Description of changes

Changelogs:
- `huggingface-hub`: https://github.com/huggingface/huggingface_hub/releases/tag/v0.22.2
- `transformers`: https://github.com/huggingface/transformers/releases/tag/v4.39.2

cc @pashashocky @happysalada 

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage commented:
Result of `nixpkgs-review pr 299198` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.farm-haystack</li>
    <li>python311Packages.farm-haystack.dist</li>
    <li>python311Packages.openllm</li>
    <li>python311Packages.openllm.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.farm-haystack</li>
    <li>python312Packages.farm-haystack.dist</li>
  </ul>
</details>
<details>
  <summary>52 packages failed to build:</summary>
  <ul>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>libretranslate (python311Packages.libretranslate)</li>
    <li>libretranslate.dist (python311Packages.libretranslate.dist)</li>
    <li>python311Packages.anthropic</li>
    <li>python311Packages.anthropic.dist</li>
    <li>python311Packages.cleanlab</li>
    <li>python311Packages.cleanlab.dist</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
    <li>python311Packages.fastembed</li>
    <li>python311Packages.fastembed.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.manifest-ml</li>
    <li>python311Packages.manifest-ml.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.vqgan-jax</li>
    <li>python311Packages.vqgan-jax.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.cleanlab</li>
    <li>python312Packages.cleanlab.dist</li>
    <li>python312Packages.diffusers</li>
    <li>python312Packages.diffusers.dist</li>
    <li>python312Packages.effdet</li>
    <li>python312Packages.effdet.dist</li>
    <li>python312Packages.fastembed</li>
    <li>python312Packages.fastembed.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio-client</li>
    <li>python312Packages.gradio-client.dist</li>
    <li>python312Packages.gradio-pdf</li>
    <li>python312Packages.gradio-pdf.dist</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
<details>
  <summary>112 packages built:</summary>
  <ul>
    <li>open-interpreter (python311Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python311Packages.open-interpreter.dist)</li>
    <li>openai-whisper (python311Packages.openai-whisper)</li>
    <li>openai-whisper.dist (python311Packages.openai-whisper.dist)</li>
    <li>python311Packages.accelerate</li>
    <li>python311Packages.accelerate.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.ctranslate2</li>
    <li>python311Packages.ctranslate2.dist</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.datasets.dist</li>
    <li>python311Packages.diffusers</li>
    <li>python311Packages.diffusers.dist</li>
    <li>python311Packages.effdet</li>
    <li>python311Packages.effdet.dist</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.evaluate.dist</li>
    <li>python311Packages.faster-whisper</li>
    <li>python311Packages.faster-whisper.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio-client</li>
    <li>python311Packages.gradio-client.dist</li>
    <li>python311Packages.gradio-pdf</li>
    <li>python311Packages.gradio-pdf.dist</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.huggingface-hub</li>
    <li>python311Packages.huggingface-hub.dist</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.litellm.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-cli.dist</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.optimum</li>
    <li>python311Packages.optimum.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.peft</li>
    <li>python311Packages.peft.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.sentence-transformers</li>
    <li>python311Packages.sentence-transformers.dist</li>
    <li>python311Packages.speechbrain</li>
    <li>python311Packages.speechbrain.dist</li>
    <li>python311Packages.stanza</li>
    <li>python311Packages.stanza.dist</li>
    <li>python311Packages.timm</li>
    <li>python311Packages.timm.dist</li>
    <li>python311Packages.tinygrad</li>
    <li>python311Packages.tinygrad.dist</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.tokenizers.dist</li>
    <li>python311Packages.torchinfo</li>
    <li>python311Packages.torchinfo.dist</li>
    <li>python311Packages.transformers</li>
    <li>python311Packages.transformers.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.accelerate</li>
    <li>python312Packages.accelerate.dist</li>
    <li>python312Packages.datasets</li>
    <li>python312Packages.datasets.dist</li>
    <li>python312Packages.evaluate</li>
    <li>python312Packages.evaluate.dist</li>
    <li>python312Packages.huggingface-hub</li>
    <li>python312Packages.huggingface-hub.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.openai-whisper</li>
    <li>python312Packages.openai-whisper.dist</li>
    <li>python312Packages.optimum</li>
    <li>python312Packages.optimum.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.peft</li>
    <li>python312Packages.peft.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.speechbrain</li>
    <li>python312Packages.speechbrain.dist</li>
    <li>python312Packages.stanza</li>
    <li>python312Packages.stanza.dist</li>
    <li>python312Packages.timm</li>
    <li>python312Packages.timm.dist</li>
    <li>python312Packages.tokenizers</li>
    <li>python312Packages.tokenizers.dist</li>
    <li>python312Packages.torchinfo</li>
    <li>python312Packages.torchinfo.dist</li>
    <li>python312Packages.transformers</li>
    <li>python312Packages.transformers.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>

a-n-n-a-l-e-e commented:
> 52 packages failed to build:
```
$ hydra-check --short --channel master  $(sed '/\.dist$/d' failed-list)
[...]
Build Status for python311Packages.k-diffusion.x86_64-linux on master
✔ python3.11-k-diffusion-0.1.1.post1 from 2024-03-26 - https://hydra.nixos.org/build/254309613
```
shows one build that succeeds on master prior to this change.

GaetanLepage commented:
Result of `nixpkgs-review pr 299198` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>30 packages marked as broken and skipped:</summary>
  <ul>
    <li>libretranslate</li>
    <li>libretranslate.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.ctranslate2</li>
    <li>python311Packages.ctranslate2.dist</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.farm-haystack</li>
    <li>python311Packages.farm-haystack.dist</li>
    <li>python311Packages.faster-whisper</li>
    <li>python311Packages.faster-whisper.dist</li>
    <li>python311Packages.libretranslate</li>
    <li>python311Packages.libretranslate.dist</li>
    <li>python311Packages.openllm</li>
    <li>python311Packages.openllm.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.farm-haystack</li>
    <li>python312Packages.farm-haystack.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>
<details>
  <summary>48 packages failed to build:</summary>
  <ul>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>python311Packages.anthropic</li>
    <li>python311Packages.anthropic.dist</li>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.cleanlab</li>
    <li>python311Packages.cleanlab.dist</li>
    <li>python311Packages.fastembed</li>
    <li>python311Packages.fastembed.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-cli.dist</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.manifest-ml</li>
    <li>python311Packages.manifest-ml.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.tinygrad</li>
    <li>python311Packages.tinygrad.dist</li>
    <li>python311Packages.torchinfo</li>
    <li>python311Packages.torchinfo.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.cleanlab</li>
    <li>python312Packages.cleanlab.dist</li>
    <li>python312Packages.effdet</li>
    <li>python312Packages.effdet.dist</li>
    <li>python312Packages.fastembed</li>
    <li>python312Packages.fastembed.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.torchinfo</li>
    <li>python312Packages.torchinfo.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
  </ul>
</details>
<details>
  <summary>94 packages built:</summary>
  <ul>
    <li>open-interpreter (python311Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python311Packages.open-interpreter.dist)</li>
    <li>openai-whisper (python311Packages.openai-whisper)</li>
    <li>openai-whisper.dist (python311Packages.openai-whisper.dist)</li>
    <li>python311Packages.accelerate</li>
    <li>python311Packages.accelerate.dist</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.datasets.dist</li>
    <li>python311Packages.diffusers</li>
    <li>python311Packages.diffusers.dist</li>
    <li>python311Packages.effdet</li>
    <li>python311Packages.effdet.dist</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.evaluate.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio-client</li>
    <li>python311Packages.gradio-client.dist</li>
    <li>python311Packages.gradio-pdf</li>
    <li>python311Packages.gradio-pdf.dist</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.huggingface-hub</li>
    <li>python311Packages.huggingface-hub.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.litellm.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.optimum</li>
    <li>python311Packages.optimum.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.peft</li>
    <li>python311Packages.peft.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.sentence-transformers</li>
    <li>python311Packages.sentence-transformers.dist</li>
    <li>python311Packages.speechbrain</li>
    <li>python311Packages.speechbrain.dist</li>
    <li>python311Packages.stanza</li>
    <li>python311Packages.stanza.dist</li>
    <li>python311Packages.timm</li>
    <li>python311Packages.timm.dist</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.tokenizers.dist</li>
    <li>python311Packages.transformers</li>
    <li>python311Packages.transformers.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.accelerate</li>
    <li>python312Packages.accelerate.dist</li>
    <li>python312Packages.datasets</li>
    <li>python312Packages.datasets.dist</li>
    <li>python312Packages.diffusers</li>
    <li>python312Packages.diffusers.dist</li>
    <li>python312Packages.evaluate</li>
    <li>python312Packages.evaluate.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio-client</li>
    <li>python312Packages.gradio-client.dist</li>
    <li>python312Packages.gradio-pdf</li>
    <li>python312Packages.gradio-pdf.dist</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.huggingface-hub</li>
    <li>python312Packages.huggingface-hub.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.openai-whisper</li>
    <li>python312Packages.openai-whisper.dist</li>
    <li>python312Packages.optimum</li>
    <li>python312Packages.optimum.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.peft</li>
    <li>python312Packages.peft.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.speechbrain</li>
    <li>python312Packages.speechbrain.dist</li>
    <li>python312Packages.stanza</li>
    <li>python312Packages.stanza.dist</li>
    <li>python312Packages.timm</li>
    <li>python312Packages.timm.dist</li>
    <li>python312Packages.tokenizers</li>
    <li>python312Packages.tokenizers.dist</li>
    <li>python312Packages.transformers</li>
    <li>python312Packages.transformers.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
  </ul>
</details>

GaetanLepage commented:
Result of `nixpkgs-review pr 299198` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>40 packages marked as broken and skipped:</summary>
  <ul>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>libretranslate</li>
    <li>libretranslate.dist</li>
    <li>openai-whisper</li>
    <li>openai-whisper.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.cleanlab</li>
    <li>python311Packages.cleanlab.dist</li>
    <li>python311Packages.ctranslate2</li>
    <li>python311Packages.ctranslate2.dist</li>
    <li>python311Packages.farm-haystack</li>
    <li>python311Packages.farm-haystack.dist</li>
    <li>python311Packages.faster-whisper</li>
    <li>python311Packages.faster-whisper.dist</li>
    <li>python311Packages.libretranslate</li>
    <li>python311Packages.libretranslate.dist</li>
    <li>python311Packages.openai-whisper</li>
    <li>python311Packages.openai-whisper.dist</li>
    <li>python311Packages.openllm</li>
    <li>python311Packages.openllm.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.cleanlab</li>
    <li>python312Packages.cleanlab.dist</li>
    <li>python312Packages.farm-haystack</li>
    <li>python312Packages.farm-haystack.dist</li>
    <li>python312Packages.openai-whisper</li>
    <li>python312Packages.openai-whisper.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>
<details>
  <summary>46 packages failed to build:</summary>
  <ul>
    <li>python311Packages.anthropic</li>
    <li>python311Packages.anthropic.dist</li>
    <li>python311Packages.fastembed</li>
    <li>python311Packages.fastembed.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.manifest-ml</li>
    <li>python311Packages.manifest-ml.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.tinygrad</li>
    <li>python311Packages.tinygrad.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.diffusers</li>
    <li>python312Packages.diffusers.dist</li>
    <li>python312Packages.effdet</li>
    <li>python312Packages.effdet.dist</li>
    <li>python312Packages.fastembed</li>
    <li>python312Packages.fastembed.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio-client</li>
    <li>python312Packages.gradio-client.dist</li>
    <li>python312Packages.gradio-pdf</li>
    <li>python312Packages.gradio-pdf.dist</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
  </ul>
</details>
<details>
  <summary>86 packages built:</summary>
  <ul>
    <li>open-interpreter (python311Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python311Packages.open-interpreter.dist)</li>
    <li>python311Packages.accelerate</li>
    <li>python311Packages.accelerate.dist</li>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.datasets.dist</li>
    <li>python311Packages.diffusers</li>
    <li>python311Packages.diffusers.dist</li>
    <li>python311Packages.effdet</li>
    <li>python311Packages.effdet.dist</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.evaluate.dist</li>
    <li>python311Packages.gradio-client</li>
    <li>python311Packages.gradio-client.dist</li>
    <li>python311Packages.gradio-pdf</li>
    <li>python311Packages.gradio-pdf.dist</li>
    <li>python311Packages.huggingface-hub</li>
    <li>python311Packages.huggingface-hub.dist</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.litellm.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-cli.dist</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.optimum</li>
    <li>python311Packages.optimum.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.peft</li>
    <li>python311Packages.peft.dist</li>
    <li>python311Packages.sentence-transformers</li>
    <li>python311Packages.sentence-transformers.dist</li>
    <li>python311Packages.speechbrain</li>
    <li>python311Packages.speechbrain.dist</li>
    <li>python311Packages.stanza</li>
    <li>python311Packages.stanza.dist</li>
    <li>python311Packages.timm</li>
    <li>python311Packages.timm.dist</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.tokenizers.dist</li>
    <li>python311Packages.torchinfo</li>
    <li>python311Packages.torchinfo.dist</li>
    <li>python311Packages.transformers</li>
    <li>python311Packages.transformers.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.accelerate</li>
    <li>python312Packages.accelerate.dist</li>
    <li>python312Packages.datasets</li>
    <li>python312Packages.datasets.dist</li>
    <li>python312Packages.evaluate</li>
    <li>python312Packages.evaluate.dist</li>
    <li>python312Packages.huggingface-hub</li>
    <li>python312Packages.huggingface-hub.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.optimum</li>
    <li>python312Packages.optimum.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.peft</li>
    <li>python312Packages.peft.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.speechbrain</li>
    <li>python312Packages.speechbrain.dist</li>
    <li>python312Packages.stanza</li>
    <li>python312Packages.stanza.dist</li>
    <li>python312Packages.timm</li>
    <li>python312Packages.timm.dist</li>
    <li>python312Packages.tokenizers</li>
    <li>python312Packages.tokenizers.dist</li>
    <li>python312Packages.torchinfo</li>
    <li>python312Packages.torchinfo.dist</li>
    <li>python312Packages.transformers</li>
    <li>python312Packages.transformers.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
  </ul>
</details>

GaetanLepage commented:
Result of `nixpkgs-review pr 299198` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>26 packages marked as broken and skipped:</summary>
  <ul>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>libretranslate</li>
    <li>libretranslate.dist</li>
    <li>openai-whisper</li>
    <li>openai-whisper.dist</li>
    <li>python311Packages.cleanlab</li>
    <li>python311Packages.cleanlab.dist</li>
    <li>python311Packages.farm-haystack</li>
    <li>python311Packages.farm-haystack.dist</li>
    <li>python311Packages.libretranslate</li>
    <li>python311Packages.libretranslate.dist</li>
    <li>python311Packages.openai-whisper</li>
    <li>python311Packages.openai-whisper.dist</li>
    <li>python311Packages.openllm</li>
    <li>python311Packages.openllm.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.cleanlab</li>
    <li>python312Packages.cleanlab.dist</li>
    <li>python312Packages.farm-haystack</li>
    <li>python312Packages.farm-haystack.dist</li>
    <li>python312Packages.openai-whisper</li>
    <li>python312Packages.openai-whisper.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
<details>
  <summary>66 packages failed to build:</summary>
  <ul>
    <li>python311Packages.anthropic</li>
    <li>python311Packages.anthropic.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.ctranslate2</li>
    <li>python311Packages.ctranslate2.dist</li>
    <li>python311Packages.fastembed</li>
    <li>python311Packages.fastembed.dist</li>
    <li>python311Packages.faster-whisper</li>
    <li>python311Packages.faster-whisper.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-cli.dist</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.manifest-ml</li>
    <li>python311Packages.manifest-ml.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.tinygrad</li>
    <li>python311Packages.tinygrad.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.diffusers</li>
    <li>python312Packages.diffusers.dist</li>
    <li>python312Packages.effdet</li>
    <li>python312Packages.effdet.dist</li>
    <li>python312Packages.fastembed</li>
    <li>python312Packages.fastembed.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio-client</li>
    <li>python312Packages.gradio-client.dist</li>
    <li>python312Packages.gradio-pdf</li>
    <li>python312Packages.gradio-pdf.dist</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>
<details>
  <summary>80 packages built:</summary>
  <ul>
    <li>open-interpreter (python311Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python311Packages.open-interpreter.dist)</li>
    <li>python311Packages.accelerate</li>
    <li>python311Packages.accelerate.dist</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.datasets.dist</li>
    <li>python311Packages.diffusers</li>
    <li>python311Packages.diffusers.dist</li>
    <li>python311Packages.effdet</li>
    <li>python311Packages.effdet.dist</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.evaluate.dist</li>
    <li>python311Packages.gradio-client</li>
    <li>python311Packages.gradio-client.dist</li>
    <li>python311Packages.gradio-pdf</li>
    <li>python311Packages.gradio-pdf.dist</li>
    <li>python311Packages.huggingface-hub</li>
    <li>python311Packages.huggingface-hub.dist</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.litellm.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.optimum</li>
    <li>python311Packages.optimum.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.peft</li>
    <li>python311Packages.peft.dist</li>
    <li>python311Packages.sentence-transformers</li>
    <li>python311Packages.sentence-transformers.dist</li>
    <li>python311Packages.speechbrain</li>
    <li>python311Packages.speechbrain.dist</li>
    <li>python311Packages.stanza</li>
    <li>python311Packages.stanza.dist</li>
    <li>python311Packages.timm</li>
    <li>python311Packages.timm.dist</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.tokenizers.dist</li>
    <li>python311Packages.torchinfo</li>
    <li>python311Packages.torchinfo.dist</li>
    <li>python311Packages.transformers</li>
    <li>python311Packages.transformers.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.accelerate</li>
    <li>python312Packages.accelerate.dist</li>
    <li>python312Packages.datasets</li>
    <li>python312Packages.datasets.dist</li>
    <li>python312Packages.evaluate</li>
    <li>python312Packages.evaluate.dist</li>
    <li>python312Packages.huggingface-hub</li>
    <li>python312Packages.huggingface-hub.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.optimum</li>
    <li>python312Packages.optimum.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.peft</li>
    <li>python312Packages.peft.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.speechbrain</li>
    <li>python312Packages.speechbrain.dist</li>
    <li>python312Packages.stanza</li>
    <li>python312Packages.stanza.dist</li>
    <li>python312Packages.timm</li>
    <li>python312Packages.timm.dist</li>
    <li>python312Packages.tokenizers</li>
    <li>python312Packages.tokenizers.dist</li>
    <li>python312Packages.torchinfo</li>
    <li>python312Packages.torchinfo.dist</li>
    <li>python312Packages.transformers</li>
    <li>python312Packages.transformers.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
  </ul>
</details>

GaetanLepage commented:
Result of `nixpkgs-review pr 299198` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.farm-haystack</li>
    <li>python311Packages.farm-haystack.dist</li>
    <li>python311Packages.openllm</li>
    <li>python311Packages.openllm.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.farm-haystack</li>
    <li>python312Packages.farm-haystack.dist</li>
  </ul>
</details>
<details>
  <summary>50 packages failed to build:</summary>
  <ul>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>libretranslate (python311Packages.libretranslate)</li>
    <li>libretranslate.dist (python311Packages.libretranslate.dist)</li>
    <li>python311Packages.anthropic</li>
    <li>python311Packages.anthropic.dist</li>
    <li>python311Packages.cleanlab</li>
    <li>python311Packages.cleanlab.dist</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
    <li>python311Packages.fastembed</li>
    <li>python311Packages.fastembed.dist</li>
    <li>python311Packages.manifest-ml</li>
    <li>python311Packages.manifest-ml.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.vqgan-jax</li>
    <li>python311Packages.vqgan-jax.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.cleanlab</li>
    <li>python312Packages.cleanlab.dist</li>
    <li>python312Packages.diffusers</li>
    <li>python312Packages.diffusers.dist</li>
    <li>python312Packages.effdet</li>
    <li>python312Packages.effdet.dist</li>
    <li>python312Packages.fastembed</li>
    <li>python312Packages.fastembed.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio-client</li>
    <li>python312Packages.gradio-client.dist</li>
    <li>python312Packages.gradio-pdf</li>
    <li>python312Packages.gradio-pdf.dist</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
<details>
  <summary>114 packages built:</summary>
  <ul>
    <li>open-interpreter (python311Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python311Packages.open-interpreter.dist)</li>
    <li>openai-whisper (python311Packages.openai-whisper)</li>
    <li>openai-whisper.dist (python311Packages.openai-whisper.dist)</li>
    <li>python311Packages.accelerate</li>
    <li>python311Packages.accelerate.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.ctranslate2</li>
    <li>python311Packages.ctranslate2.dist</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.datasets.dist</li>
    <li>python311Packages.diffusers</li>
    <li>python311Packages.diffusers.dist</li>
    <li>python311Packages.effdet</li>
    <li>python311Packages.effdet.dist</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.evaluate.dist</li>
    <li>python311Packages.faster-whisper</li>
    <li>python311Packages.faster-whisper.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio-client</li>
    <li>python311Packages.gradio-client.dist</li>
    <li>python311Packages.gradio-pdf</li>
    <li>python311Packages.gradio-pdf.dist</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.huggingface-hub</li>
    <li>python311Packages.huggingface-hub.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.litellm.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-cli.dist</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.optimum</li>
    <li>python311Packages.optimum.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.peft</li>
    <li>python311Packages.peft.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.sentence-transformers</li>
    <li>python311Packages.sentence-transformers.dist</li>
    <li>python311Packages.speechbrain</li>
    <li>python311Packages.speechbrain.dist</li>
    <li>python311Packages.stanza</li>
    <li>python311Packages.stanza.dist</li>
    <li>python311Packages.timm</li>
    <li>python311Packages.timm.dist</li>
    <li>python311Packages.tinygrad</li>
    <li>python311Packages.tinygrad.dist</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.tokenizers.dist</li>
    <li>python311Packages.torchinfo</li>
    <li>python311Packages.torchinfo.dist</li>
    <li>python311Packages.transformers</li>
    <li>python311Packages.transformers.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.accelerate</li>
    <li>python312Packages.accelerate.dist</li>
    <li>python312Packages.datasets</li>
    <li>python312Packages.datasets.dist</li>
    <li>python312Packages.evaluate</li>
    <li>python312Packages.evaluate.dist</li>
    <li>python312Packages.huggingface-hub</li>
    <li>python312Packages.huggingface-hub.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.openai-whisper</li>
    <li>python312Packages.openai-whisper.dist</li>
    <li>python312Packages.optimum</li>
    <li>python312Packages.optimum.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.peft</li>
    <li>python312Packages.peft.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.speechbrain</li>
    <li>python312Packages.speechbrain.dist</li>
    <li>python312Packages.stanza</li>
    <li>python312Packages.stanza.dist</li>
    <li>python312Packages.timm</li>
    <li>python312Packages.timm.dist</li>
    <li>python312Packages.tokenizers</li>
    <li>python312Packages.tokenizers.dist</li>
    <li>python312Packages.torchinfo</li>
    <li>python312Packages.torchinfo.dist</li>
    <li>python312Packages.transformers</li>
    <li>python312Packages.transformers.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>

GaetanLepage commented:
> shows one package in the failing list that succeeds on master without this change.

Looks like it was just "bad luck". I re-ran `nixpkgs-review` and it has built fine this time.

a-n-n-a-l-e-e commented:
> > shows one package in the failing list that succeeds on master without this change.
> 
> Looks like it was just "bad luck". I re-ran `nixpkgs-review` and it has built fine this time.

ok. what about the other archs? the change looks simple enough but i can't merge a commit from a package that is showing so many broken packages. can use `hydra-check` from nixpkgs with `--arch` arg to specify other systems to verify they are broken on master. thx.

GaetanLepage committed:
python311Packages.huggingface-hub: 0.22.0 -> 0.22.2

Diff: https://github.com/huggingface/huggingface_hub/compare/refs/tags/v0.22.0...v0.22.2

Changelog: https://github.com/huggingface/huggingface_hub/releases/tag/v0.22.2

GaetanLepage committed:
python311Packages.transformers: 4.39.1 -> 4.39.2

Diff: https://github.com/huggingface/transformers/compare/refs/tags/v4.39.1...v4.39.2

Changelog: https://github.com/huggingface/transformers/releases/tag/v4.39.2

happysalada commented:
@a-n-n-a-l-e-e I'm happy to take the responsability to merge here if you don't mind.
Transformers has been known to have several broken packages associated with it. The last updates were exactly like that.

a-n-n-a-l-e-e commented:
> @a-n-n-a-l-e-e I'm happy to take the responsability to merge here if you don't mind. Transformers has been known to have several broken packages associated with it. The last updates were exactly like that.

go for it!

GaetanLepage commented:
Result of `nixpkgs-review pr 299198` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.farm-haystack</li>
    <li>python311Packages.farm-haystack.dist</li>
    <li>python311Packages.openllm</li>
    <li>python311Packages.openllm.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.farm-haystack</li>
    <li>python312Packages.farm-haystack.dist</li>
  </ul>
</details>
<details>
  <summary>32 packages failed to build:</summary>
  <ul>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>python311Packages.anthropic</li>
    <li>python311Packages.anthropic.dist</li>
    <li>python311Packages.cleanlab</li>
    <li>python311Packages.cleanlab.dist</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
    <li>python311Packages.fastembed</li>
    <li>python311Packages.fastembed.dist</li>
    <li>python311Packages.vqgan-jax</li>
    <li>python311Packages.vqgan-jax.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.cleanlab</li>
    <li>python312Packages.cleanlab.dist</li>
    <li>python312Packages.diffusers</li>
    <li>python312Packages.diffusers.dist</li>
    <li>python312Packages.effdet</li>
    <li>python312Packages.effdet.dist</li>
    <li>python312Packages.fastembed</li>
    <li>python312Packages.fastembed.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio-client</li>
    <li>python312Packages.gradio-client.dist</li>
    <li>python312Packages.gradio-pdf</li>
    <li>python312Packages.gradio-pdf.dist</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
<details>
  <summary>138 packages built:</summary>
  <ul>
    <li>libretranslate (python311Packages.libretranslate)</li>
    <li>libretranslate.dist (python311Packages.libretranslate.dist)</li>
    <li>open-interpreter (python311Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python311Packages.open-interpreter.dist)</li>
    <li>openai-whisper (python311Packages.openai-whisper)</li>
    <li>openai-whisper.dist (python311Packages.openai-whisper.dist)</li>
    <li>python311Packages.accelerate</li>
    <li>python311Packages.accelerate.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.chromadb</li>
    <li>python311Packages.chromadb.dist</li>
    <li>python311Packages.ctranslate2</li>
    <li>python311Packages.ctranslate2.dist</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.datasets.dist</li>
    <li>python311Packages.diffusers</li>
    <li>python311Packages.diffusers.dist</li>
    <li>python311Packages.effdet</li>
    <li>python311Packages.effdet.dist</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.evaluate.dist</li>
    <li>python311Packages.faster-whisper</li>
    <li>python311Packages.faster-whisper.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio-client</li>
    <li>python311Packages.gradio-client.dist</li>
    <li>python311Packages.gradio-pdf</li>
    <li>python311Packages.gradio-pdf.dist</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.huggingface-hub</li>
    <li>python311Packages.huggingface-hub.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.litellm.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-cli.dist</li>
    <li>python311Packages.llama-index-embeddings-huggingface</li>
    <li>python311Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python311Packages.llama-index-llms-openai-like</li>
    <li>python311Packages.llama-index-llms-openai-like.dist</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.manifest-ml</li>
    <li>python311Packages.manifest-ml.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.optimum</li>
    <li>python311Packages.optimum.dist</li>
    <li>python311Packages.outlines</li>
    <li>python311Packages.outlines.dist</li>
    <li>python311Packages.peft</li>
    <li>python311Packages.peft.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.sentence-transformers</li>
    <li>python311Packages.sentence-transformers.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.speechbrain</li>
    <li>python311Packages.speechbrain.dist</li>
    <li>python311Packages.stanza</li>
    <li>python311Packages.stanza.dist</li>
    <li>python311Packages.timm</li>
    <li>python311Packages.timm.dist</li>
    <li>python311Packages.tinygrad</li>
    <li>python311Packages.tinygrad.dist</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.tokenizers.dist</li>
    <li>python311Packages.torchinfo</li>
    <li>python311Packages.torchinfo.dist</li>
    <li>python311Packages.transformers</li>
    <li>python311Packages.transformers.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.accelerate</li>
    <li>python312Packages.accelerate.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.datasets</li>
    <li>python312Packages.datasets.dist</li>
    <li>python312Packages.evaluate</li>
    <li>python312Packages.evaluate.dist</li>
    <li>python312Packages.huggingface-hub</li>
    <li>python312Packages.huggingface-hub.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.openai-whisper</li>
    <li>python312Packages.openai-whisper.dist</li>
    <li>python312Packages.optimum</li>
    <li>python312Packages.optimum.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.peft</li>
    <li>python312Packages.peft.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.speechbrain</li>
    <li>python312Packages.speechbrain.dist</li>
    <li>python312Packages.stanza</li>
    <li>python312Packages.stanza.dist</li>
    <li>python312Packages.timm</li>
    <li>python312Packages.timm.dist</li>
    <li>python312Packages.tokenizers</li>
    <li>python312Packages.tokenizers.dist</li>
    <li>python312Packages.torchinfo</li>
    <li>python312Packages.torchinfo.dist</li>
    <li>python312Packages.transformers</li>
    <li>python312Packages.transformers.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>
